shared/reducers/waiting.js

```diff
@@ -10,11 +10,23 @@ function reducer(state: Types.State = Constants.initialState, action: Waiting.Ac
     }
     case Waiting.decrementWaiting: {
       const {payload: {key}} = action
-      return state.update(key, 1, n => n - 1)
+      if (typeof key === 'string') {
```

Comment:
could just detect if its a single string and just convert it to [key] and keep the rest the same